docs/contributor/04-30-ci-cd.md

```diff
@@ -73,6 +75,7 @@ The following CI jobs generate release artifacts and verify the functional corre
 
 The following CI jobs are scheduled to run at specific times.
 
-| Name                                                                                                                                                                                  | Description                                                                             |
-|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| Name                                                                                                                                                        | Description                                                                             |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
 | [`Istio performance test`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/performance-test.yaml#L10) | Runs Istio Manager's performance tests every day at 12:00 AM on a Gardener GCP cluster. |
+| [`UI Tests Periodic / Run UI tests`]()                                                                                                                      | Runs UI tests for Busola extensions of Istio Module.                                    |
```

Comment:
For api-gateway docs we also listed `UI Tests Periodic / Slack Notification`. I'm not sure if we need this.

Reply:
We have it listed for main integration workflow. To stay consistent I also listed it here.

Reply:
Yeah, let's add it.

Reply:
We have it in the main integration part, so we stay consistent.